### PR TITLE
scripts: fix stork in-repo check. add clang-format files.

### LINKED ISSUE #9064: Make following clang format easy for new developers (Stork)
### Description of the enhancement or error report

If a new user "forks" Stork, it's several more manual steps to get clang format up and running if they want to use commit hooks and our clang format file. This should be easier.

### Rationale for the enhancement or information for reproducing the error

New users can't immediately use the clang format tool without several manual steps.

### Identified impact
(i.e. Internal object changes, limited interface changes, public API change, or a list of specific applications impacted)

Better experience for new users.

### PR DESCRIPTION
**Pull request by rwcarlsen**
Fix bug where stork.sh script failed to recognize when it was in the
MOOSE (or any other) git repository. stork.sh now adds .clang-format and
pre-commit hook install script files to new users' app repositories.

fixes #9064

### CODE REVIEW

File: scripts/stork.sh [hunk 1]
```diff
@@ -85,11 +83,14 @@ if [[ "$kind" == "module" ]]; then
 fi
 
 if [[ "$kind" == "app" ]]; then
-    (
-        cd $dir
-        git init
-        git add *
-    )
+    # copy clang-format related files
+    mkdir -p $dir/scripts
```
Comment (permcody) [inline scripts/stork.sh]: I know this isn't the first place in this script where you "write" but what does this script do if you don't have write permissions where you are running (trust me, users do this....). Perhaps that should be added as another up front check?

File: scripts/stork.sh [hunk 2]
```diff
@@ -100,5 +101,11 @@ if [[ "$kind" == "app" ]]; then
     echo "         git remote add origin https://github.com/YourGitHubUserName/$dstname"
     echo '         git commit -m "initial commit"'
     echo "         git push -u origin master"
+    echo ""
+    echo "To automatically enforce MOOSE C++ code style in your commits, run:"
```
Comment (permcody) [inline scripts/stork.sh]: 👍

**Review by permcody — CHANGES_REQUESTED** [review]

File: scripts/stork.sh (hunk 1 above)
Comment (rwcarlsen) [inline scripts/stork.sh]: I added an earlier abort that should cover this failure mode.

**Comment by friedmud** [comment]
I used this this weekend and it seems like it fails to add a .gitignore as
well.

Other than that: worked great!

**Comment by rwcarlsen** [comment]
good catch - .gitignore is now added.

**Comment by friedmud** [comment]
Cool - thanks!

**Review by permcody — APPROVED** [review]

**Pull request merged**